FreeRTOS/Demo/WIN32-MSVC-Static-Allocation-Only/main.c

```diff
@@ -86,6 +86,9 @@ static void prvCheckTask( void *pvParameters );
 
 int main( void )
 {
+	/* No buffer for stdout. */
+    setvbuf(stdout, NULL, _IONBF, 0);

```

Comment:
Formatting issue, need spaces after a '(' character and before a ')' character

Reply:
Add the function in git workflow file instead of demo main.c file